mesheryctl/cmd/perf.go

```diff
@@ -54,13 +75,22 @@ func StringWithCharset(length int) string {
 // perfCmd represents the Performance command
 var perfCmd = &cobra.Command{
 	Use:   "perf",
-	Short: "Performance testing and benchmarking",
-	Long:  `Performance testing and benchmarking.`,
+	Short: "Performance Testing",
+	Long:  `Performance Testing & Benchmarking using Meshery CLI.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		//Check prerequisite
 		preReqCheck()
 
-		println("Test name used : ", testName)
+		if len(args) == 0 {
+			fmt.Println(perfDetails)
```

Comment:
What benefit will log dependency give over here ? It would be better if you could tell me why is log.Info a better way than fmt.Println ?

Reply:
I am not sure that we store logs currently but will surely do in future so that will be usefull, and as we are already using logrus on massively in project for logging either its for terminal logging or using it in storing. Logrus is much more flexible then the fmt for me. 
It would be great to have @leecalcote suggestion on this.

Reply:
I dont see any issue addressing storing of logs of mesheryctl. Logrus is specifically used for logging specification purpose for Go-CLI. Commands. Since this is not a use-case over here, I dont think replacing println with a third party logger command would make any helpful change.

Reply:
I think the strongest argument here is consistency of implementation - maintaining uniformity in approach to logging throughout the project.